allennlp/experiments/registry.py

```diff
@@ -263,6 +270,80 @@ def get_initializer(cls, name) -> Callable[[torch.Tensor], None]:
             "sparse": torch.nn.init.sparse,
             "eye": torch.nn.init.eye,
     }
-    #: This decorator adds a :func:`initializer` to the registry, with the given name.
+    #: This decorator adds an ``initializer`` to the registry, with the given name.
     register_initializer = _registry_decorator("initializer", _initializers)
     default_initializer = "normal"
+
+    @classmethod
+    def list_initializers(cls) -> List[str]:
+        """
+        Returns a list of all currently-registered initializer names.
+        """
+        return _get_keys_with_default(cls._initializers, "initializer", cls.default_initializer)
+
+    @classmethod
+    def get_initializer(cls, name) -> Callable[[torch.Tensor], None]:
+        """
+        Returns the initializer that has been registered with ``name``.
+        """
+        return cls._initializers[name]
+
+    ###########################
+    # Module-related registries
+    ###########################
+
+    # Token Vectorizers
+
+    _token_vectorizers = {}  # type: Dict[str, Type[TokenVectorizer]]
+    #: This decorator adds a :class:`TokenVectorizer` to the regsitry, with the given name.
```

Comment:
Yeah, I was working from an older commit, before you fixed the typos...  Will fix.